src/main/scala/nl.knaw.dans.easy.bagstore/component/BagProcessingComponent.scala

```diff
@@ -79,7 +79,7 @@ trait BagProcessingComponent extends DebugEnhancedLogging {
 
     // FIXME: Only called in BagStoreComp and in previous method
     def setPermissions(bagDir: Path, filePermissions: JSet[PosixFilePermission], directoryPermissions: JSet[PosixFilePermission], includeTopDir: Boolean = true) = Try {
-      logger.info(s"Setting bag permissions to: file = $filePermissions, dir = $directoryPermissions, bag directory: $bagDir")
+      logger.debug(s"Setting bag permissions to: file = $filePermissions, dir = $directoryPermissions, bag directory: $bagDir")
```

Comment:
This does not need to be info. Only when diagnosing problems do we need this kind of information.